miner/scroll_worker.go

```diff
@@ -292,13 +292,40 @@ func (w *worker) close() {
 	w.wg.Wait()
 }
 
+// checkHeadRowConsumption will start some initial workers to CCC check block close to the HEAD
+func (w *worker) checkHeadRowConsumption() error {
+	checkBoundry := uint64(1)
+	numOfBlockToCheck := uint64(w.config.CCCMaxWorkers + 1)
```

Comment:
What about adding a constant in case of `w.config.CCCMaxWorkers` changes?

Reply:
it is fine if w.config.CCCMaxWorkers increases but it is a bit risky to reduce it.Let me see if I can find a not overly complicated fix.

Reply:
agree. it does not matter in most cases. just an edge case.